src/resources/views/errors/403.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends ('layouts.default')
+
+@section ('page_title', '403 - Forbidden')
+
+@section ('content')
+
+<div class="container pt-1">
+
+    <div class="pb-2 mt-4 mb-4 border-bottom">
+        <h1>
+            Oops - 403 - Forbidden
```

Comment:
localization :)

Reply:
Just copied that from the 404.blade.php, modified the message obviously.